src/core/compiler.js

```diff
@@ -62,11 +74,11 @@ const setValues = (input, entries) => defineProperty(input, VALUES, entries);
 export const getValues = (input) => input[VALUES];
 
 export const matchedBy = (searches) => Array
-  .from(new Set([].concat(...searches.map((e) => getLeafNeedlesMatch(e)))));
```

Comment:
Apparently, this syntax uses recursion - at least I was getting `RangeError: Maximum call stack size exceeded`.

`flatMap` seems to be the way to go (and hopefully more performant)